packages/squiggle-lang/src/rescript/Reducer/Reducer_MathJs/Reducer_MathJs_ToCodeTree.res

```diff
@@ -0,0 +1,111 @@
+module CTT = Reducer_CodeTree_T
```

Comment:
Btw, have you considered adding Interface files? We haven't used them much yet, but for this work, they could help keep things clean.
https://rescript-lang.org/docs/reason-compiler/latest/automatic-interface-generation

Reply:
I'm not unhappy with the pattern of having files that consist of nothing but module aliases. 

on the `reducer/README.md` file I made a suggestion to leverage interface files to enforce that reducer internals aren't messed with

Reply:
I am following https://dev.to/yawaramin/a-modular-ocaml-project-structure-1ikd
Because of that there are directory modules that have only module aliases